doc/source/whatsnew/v0.24.0.txt

```diff
@@ -214,6 +214,7 @@ Other Enhancements
 - Compatibility with Matplotlib 3.0 (:issue:`22790`).
 - Added :meth:`Interval.overlaps`, :meth:`IntervalArray.overlaps`, and :meth:`IntervalIndex.overlaps` for determining overlaps between interval-like objects (:issue:`21998`)
 - :meth:`Timestamp.tz_localize`, :meth:`DatetimeIndex.tz_localize`, and :meth:`Series.tz_localize` have gained the ``nonexistent`` argument for alternative handling of nonexistent times. See :ref:`timeseries.timezone_nonexsistent` (:issue:`8917`)
+- :meth:`Styler.pipe` method added, to simplify application of user-defined functions that operate on stylers.
```

Comment:
Reference the issue number.

Reply:
Also, you should create a mini-section explaining the enhancement in more detail so that end users can understand the benefit.